app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -170,16 +206,39 @@ def with_disk_attachments_service(vm)
   end
 
   class_methods do
+    def api3_supported_features
+      []
+    end
+
+    def api4_supported_features
+      [:reconfigure_vms, :snapshots]
```

Comment:
"snapshots" existed before - [](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/vmware/infra_manager/vm.rb#L27)

as for reconfigure_vms its a bad name, it will actually have a different name, its for another PR, I will just remove it from here. Some providers allow one to reconfigure disks during vm reconfiguration (VMWare, and now RHV) and some do not.
Same goes for quick_stats (also another PR) up until now it was supported only for VMWare, but now that RHV also supports it we need to name it. see [](https://github.com/ManageIQ/manageiq/pull/10505/files#diff-cae72f3c7d5b9ebc549497c6d8e9834aR1311)